modules/nw-dpu-installing-operator.adoc

```diff
@@ -0,0 +1,162 @@
+
+. Create a `Subscription` CR for the SR-IOV Network Operator by entering the following command:
++
+[source,terminal]
+----
+$ cat << EOF| oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-dpu-operator-subscription
+  namespace: openshift-dpu-operator
+spec:
+  channel: stable
+  name: dpu-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.Verification
+
+. Check that the Operator is installed by entering the following command:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-dpu-operator \
+  -o custom-columns=Name:.metadata.name,Phase:.status.phase
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+Name                                Phase
+dpu-operator.v4.18.0-202503130333   Succeeded
+----
+
+. Change to the `openshift-dpu-operator` project:
++
+[source,terminal]
+----
+$ oc project openshift-dpu-operator
+----
+
+. Verify the DPU Operator is running by entering the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-dpu-operator
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+NAME                                               READY   STATUS    RESTARTS   AGE
+dpu-operator-controller-manager-6b7bbb5db8-7lvkj   2/2     Running   0          2m9s
+----
+
+[id="install-dpu-operator-web-console_{context}"]
+== Installing the DPU Operator using the web console
+
+As a cluster administrator, you can install the DPU Operator by using the web console.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* An account with `cluster-admin` privileges.
+
+.Procedure
+
+. Install the DPU Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Select *DPU Operator* from the list of available Operators, and then click *Install*.
+
+.. On the *Install Operator* page, under *Installed Namespace*, the *Operator recommended Namespace* option is preselected by default. No action is required..
+
+.. Click *Install*.
+
+. Verify that the DPU Operator is installed successfully:
+
+.. Navigate to the *Operators* -> *Installed Operators* page.
+
+.. Ensure that *DPU Operator* is listed in the *openshift-dpu-operator* project with a *Status* of *InstallSucceeded*.
++
+[NOTE]
+====
+During installation an Operator might display a *Failed* status.
+If the installation later succeeds with an *InstallSucceeded* message, you can ignore the *Failed* message.
+====
++
+If the Operator does not appear as installed, to troubleshoot further:
++
+* Inspect the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors under *Status*.
+
+* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the `openshift-dpu-operator` project.
+
+* Check the namespace of the YAML file. If the annotation is missing, you can add the annotation `workload.openshift.io/allowed=management` to the Operator namespace with the following command:
++
```

Comment:
We need to add that all the nodes in the cluster have a dpu should be labeled with `dpu=true`. example:

`oc label node worker-1 dpu=true`

Reply:
Also,  we need the following as Prerequisites:

```
Prerequisites

- An OpenShift cluster installed on the host system
- A MicroShift cluster installed on the DPUs
- Access to the cluster as a user with the cluster-admin role.
- The OpenShift CLI (oc) installed.

```